### ISSUE TITLE
Text is not in the selected Language when user finished speaking.

### ISSUE THREAD

**Issue by haiderali02** [AppDevGuy/OSSSpeechKit#23]
Hi, It's working perfectly in English language and quite accurate. But when I switch to other language i.e Danish only the speaker changed and when I finish speaking. It's return me text in English but not in Danish. Please kindly review.

**Comment by AppDevGuy** [comment]
Hi @haiderali02 - I am unsure if you are trying to:

- Speak Danish and have it return Danish text or
- Speak English and have it return Danish text.

SpeechKit unfortunately does not do translation. Apple have [not made available their translation service API available yet](https://developer.apple.com/forums/thread/650024) either despite offering the new app for it. If you are hoping to have the speech _translated across languages_ this SDK is not the right one to use. Google do however offer a translation service. 

If however you are having issues with Dutch speech being translated to Dutch text, let me know and I will have a look.

**Comment by haiderali02** [comment]
Hi, Thanks for your reply, I am speaking in Danish but in return it should
display me or return me the danish text.

It is not doing in that way. I speak danish and it return me English
language strings

**Comment by AppDevGuy** [comment]
Thanks for the prompt reply. Could you please share the code where you initialise OSSSpeechKit?

The short answer is to make sure you have set the correct voice when speaking: 

```
speechKit.voice = OSSVoice(quality: .enhanced, language: . Danish)
```

If it is set to an English language it will always attempt to return in English.

**Comment by haiderali02** [comment]
Here is my code.

// Declare an instance of OSSSpeechKit
    let speechKit = OSSSpeech.shared
    // Create a voice instance
    let newVoice = OSSVoice()
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        speechToDutch()
        // Do any additional setup after loading the view.
    }


    // MARK:- Class functions -
    
    func speechToDutch () {
        speechText.text = "Hallo, ik ben Lianna"
    
        self.cTitle.text = "Toespraak naar tekst"
        
        newVoice.language = OSSVoiceEnum.Danish.rawValue
        // Set the voice quality
        newVoice.quality = .enhanced
        // Set the voice of the speech kit
        speechKit.voice = newVoice
        speechKit.accessibilityLanguage = "da_DK"
        
        // Initialise an utterance
        let utterance = OSSUtterance(string: "Hallo, ik ben Eva")
        // Set the recognition task type
        speechKit.recognitionTaskType = .undefined
        // Set volume
        utterance.volume = 0.5
        // Set rate of speech
        utterance.rate = 0.5
        // Set the pitch
        utterance.pitchMultiplier = 1.2
        // Set speech utterance
        speechKit.utterance = utterance
        // Ask to speak
        speechKit.speakText(utterance.speechString)
        
        speechKit.delegate = self
    }

**Comment by AppDevGuy** [comment]
Thanks. I will take a look at what the issue is now and let you know.

**Comment by haiderali02** [comment]
Thanks @AppDevGuy

**Comment by AppDevGuy** [comment]
@haiderali02 - good catch. I have found the issue and will push up a fix shortly.

**Comment by haiderali02** [comment]
Thanks @AppDevGuy , Really appreciated. :)

**Comment by AppDevGuy** [comment]
Hi @haiderali02 - please checkout `pod OSSSpeechKit, '~> 0.3.2'` for the solution. If you have any further issues let me know.

**Comment by haiderali02** [comment]
Please make available pod OSSSpeechKit, '~> 0.3.2' for install via cocopods. 
Thanks ;)

**Comment by AppDevGuy** [comment]
It should be available soon - may take anywhere from a few minutes to a few hours for Cocoapods to update but it is listed there now. Good luck and thanks for using my library.

https://cocoapods.org/pods/OSSSpeechKit

**Comment by haiderali02** [comment]
Thanks for your help and excellent support. 
Wish you Good Luck!

**Comment by AppDevGuy** [comment]
Hi @haiderali02 did v0.3.2 solve the issue you were having? If so I will close the issue.

**Comment by haiderali02** [comment]
Hi, yes issue is resolved now in v0.3.2. You may close it now. 
Thanks again.

**Comment by AppDevGuy** [comment]
Issue resolved in 0.3.2. Closing.
